ens/README.md

```diff
@@ -0,0 +1,3 @@
+# ENS
+
+NOTE: At the time of writing, the ens substream only works for ENS registered with ETH Base Registrar contract.
```

Comment:
You should add some description on what this module is for. Would also be helpful to add it into the PR description section